l10_ir_accounting/views/account_group_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="accoutn_group_action" model="ir.actions.act_window">
+            <field name="name">گروه حساب</field>
```

Comment:
use English language in the code but i18n/ir.po

Reply:
File added